spotify_player/src/config/theme.rs

```diff
@@ -58,6 +58,9 @@ pub struct Palette {
     pub bright_blue: Color,
     #[serde(default = "Color::bright_yellow")]
     pub bright_yellow: Color,
+
+    #[serde(default = "Color::dark_gray")]
+    pub dark_gray: Color,
```

Comment:
I added support for dark gray coloring, but is there a better way to add support for coloring modifications? Dark and Bright colors seem to have some repetition. And should other dark colors be added as well?

Reply:
No you shouldn't add new field to the palette, it represents the basic 16-color palette with additional fields for background and foreground.